YoutubeExplode/YoutubeClient.cs

```diff
@@ -206,6 +206,12 @@ public YoutubeClient()
                 throw new VideoNotAvailableException(errorCode, errorReason);
             }
 
+            // Check for paid content
+            if (videoInfoDic.ContainsKey("requires_purchase") && videoInfoDic.Get("requires_purchase") == "1")
```

Comment:
You only need one check:
`if (videoInfoDic.GetOrDefault("requires_purchase") == "1")`